src/components/templates/Playa/Playa.tsx

```diff
@@ -131,7 +131,7 @@ const isPlaced = (venue: Venue) => {
 const minZoom = () => (window.innerWidth - 2 * PLAYA_MARGIN_X) / PLAYA_WIDTH;
 
 const Playa = () => {
-  useFirestoreConnect("venues");
+  useSparkleFirestoreConnect("venues");
```

Comment:
[OOS] Not relevant to this PR, but calling this out as a pattern we want to remove, as we shouldn't ever be loading all of the venues like this.